client/styles/modal.scss

```diff
@@ -18,12 +20,11 @@
         @include popout();
         position: relative;
         min-width: 380px;
-        max-width: 800px;
-        max-height: 96%;
+        max-width: 440px;
+        max-height: 559px;
```

Comment:
If we can put these in a local styles file that is scoped to the specific modal(s) being affected, that would be great